rfc/rfc004-package-log.md

```diff
@@ -12,8 +12,21 @@ http://peter.bourgon.org/go-kit/#package-log
 
 ## Scope
 
-To be defined.
+* Log SHALL be drop in compatible with the standard library logger.
+* Log SHALL use key / value pairs for structured data.
```

Comment:
Definitely a use case for per log call k/v pairs in an ad-hoc fashion.  Something like .WithField / .WithFields from logrus would be nice.